README.md

```diff
@@ -127,8 +127,7 @@ To automate the formatting, we use pre-commit hooks from the
 make precommit-install
 # to run the hooks:
 make precommit
-# alternatively, running black on all staged files:
-make black  # or simply black $(git diff --staged --name-only)
+# Alternatively, run `ruff format`
```

Comment:
Should this have a command below it?

Reply:
I thought the inline command was enough, do you think it would be clearer below?

Reply:
```suggestion
# Alternatively, run ruff directly:
ruff format
```